lua/luaGlobal.h

```diff
@@ -41,11 +42,37 @@ static int hslua_g_printMessage(lua_State* lua)
     return 0;
 }
 
+static int hslua_g_printToScreen(lua_State* lua)
+{
+    std::string message = std::string();
+    int argCount = lua_gettop(lua);
+    lua_getglobal(lua, "tostring");
+    for (int i = 1; i <= argCount; i++)
+    {
+        lua_pushvalue(lua, -1);
+        lua_pushvalue(lua, i);
+        lua_call(lua, 1, 1);
+        const char* str = lua_tostring(lua, -1);
+        if (str == NULL)
+        {
+            return luaL_error(lua, "'tostring' must return a string to 'print'");
+        }
+        
+        message += str;
+        message += "    ";
+        lua_pop(lua, 1);
+    }
+    PrintHelper::GetInstance()->AddMessage(message);
+    printf("%s\n", message.c_str());
+    return 0;
+}
+
 // TODO: Maybe restructure this as a class and have this be part of constructor?
 // TODO: Or maybe constructor calls the global singleton of LuaScriptInit instance and gets the lua state?
 void hsluaglobal_register(lua_State* lua)
 {
     lua_register(lua, "log", hslua_g_printMessage);
+    lua_register(lua, "print", hslua_g_printToScreen);
```

Comment:
`print` already exists as a word if I remember right.
This is technically overwriting the existing `print` method from Lua that writes to the standard out.

Technically we probably should make it write to the hs log (and standard out) like log already does and like we probably should have made print do in the first place.